pkg/cmd/cli/backuplocation/set.go

```diff
@@ -51,14 +53,18 @@ func NewSetCommand(f client.Factory, use string) *cobra.Command {
 
 type SetOptions struct {
 	Name                         string
+	Credential                   flag.Map
 	DefaultBackupStorageLocation bool
 }
 
 func NewSetOptions() *SetOptions {
-	return &SetOptions{}
+	return &SetOptions{
+		Credential: flag.NewMap(),
+	}
 }
 
 func (o *SetOptions) BindFlags(flags *pflag.FlagSet) {
+	flags.Var(&o.Credential, "credential", "Sets the one credential to be used by this location in key-value pair, where key is the secret name, and value is the secret key name. Optional.")
```

Comment:
Hm, I wonder if we could have shared string consts for these help messages. I wouldn't say redo them all in this PR, but maybe it's something we could work towards, as repeating this in 2 files seems prone to error.